### DISCUSSION TITLE
Response time for support for Individual and Organization accounts.

### DISCUSSION THREAD

**Discussion by creativeedge** [community/community#139279] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

What is the normal response time for standard support for individual and paid Organization accounts?

Created 2 tickets.  Have not gottent a response for almost a week now.

**Comment by alx-mp4** [comment, marked as answer]
Hi @creativeedge, no need to worry! They typically respond fairly quickly. However, I recently had a ticket open, and it took them about two weeks to get back to me. A GitHub support team member explained that they've been experiencing a high volume of support requests over the past few weeks, which is causing some delays.

**Reply by creativeedge** [reply]
Thanks for responding @AleksandarHaralanov.  Finally got a response.  I appreciate your help.

**Reply by alx-mp4** [reply]
You're welcome, hope everything gets resolved asap!

**Comment by Hruthik0x** [comment]
Its been over two weeks, I have raised 2 tickets, and I did not receive any response to my tickets. Please respond to my tickets
